src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -132,6 +133,14 @@ errors::eType CObjectSchemaItem::validate(
         std::string validation_info = "Missing mandatory parameter: " + key;
         report->set_validation_info(validation_info);
         return errors::MISSING_MANDATORY_PARAMETER;
+      } else if (key.compare(msg_params) == 0) {
+        // If the message params struct was filtered, that means that the
+        // app's version is too low to use the message.
+        std::string validation_info =
```

Comment:
SubscribeWayPoints is an example of an RPC with no params. Will this block SubscribeWayPoints?

Reply:
I tested SubscribeWayPoints and it was successful

Reply:
Yeah, `msg_params` should still exist in that case, it would just be empty